buttons-examples/iOS/ios-FAB.md

```diff
@@ -152,80 +152,204 @@ in the spec.
+**Stroke width** |  | `setBorderWidth:forState:`<br/>`borderWidthForState:` | `0`
+**Ripple color** | `inkColor`   | `setInkColor`<br/>`inkColor` | White at 20% opacity
 
- to do:
-Add Shrine theming example
+#### Icon attributes
 
+&nbsp;                                          | Attribute         | Related method(s)                                         | Default value
+----------------------------------------------- | ----------------- | --------------------------------------------------------- | -------------
+**Icon**                                        | `imageView`        | `setImage:forState:`<br/>`imageForState:` | `nil`
+**Color**                                       | `imageView.tintColor` | `setImageViewTintColor:forState:`<br/>`imageViewTintColorForState:` | `nil`
 
-## Theming buttons
+#### Text label attributes
 
-FABs support [Material Theming](https://material.io/components/buttons/#theming) and can be customized in terms of color, typography and shape.
+&nbsp;         | Attribute                | Related method(s)                 | Default value
+-------------- | ------------------------ | --------------------------------- | -------------
+**Text label** | `titleLabel` | `setTitle:forState:`<br/>`titleForState:` | `nil`
+**Color**      | `titleLabel.textColor` | `setTitleColor:forState:`<br/>`titleColorForState:` | `UIColor.blackColor`
+**Typography** | `titleLabel.font` | `setFont:` and `font` on `titleLabel` | Button
+
+## Theming
+
+You can theme an MDCFloatingButton to have a secondary theme using the MDCFloatingButton theming
+extension. [Learn more about theming extensions and container schemes](../../docs/theming.md).
 
 ### FAB theming example
 
-API and source code:
+To make use of the theming methods shown in the examples above do the following:
 
-* `MaterialButton` (a subclass of [UIButton](https://developer.apple.com/documentation/uikit/uibutton))
-    * [Class description](https://)
-    * [GitHub source](https://github.com/material-components/)
-    
-The following example shows a FAB with Material Theming.
+1. Install the theming extensions with Cocoapods
+    Add the following line to your `Podfile`:
 
-!["FAB theming examples for iOS with pink and black FAB"](assets/FAB-theming.png)
+    ```
+    pod MaterialComponents/Buttons+Theming
+    ```
+    
+    Run the installer:
+    
+    ```
+    pod install
+    ```
 
-[Shrine theme](https://material.io/design/material-studies/shrine.html)
+1. Import the Buttons theming target
 
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialButtons_Theming
+```
+#### Objective-C
+```objc
+#import "MaterialButtons.h"
+#import <MaterialComponents/MaterialButtons+Theming.h>
 ```
-Include source code implementing FAB with a plus icon using "Shrine" theme.
 
-Upload a screenshot of the render and update the image.
```

Comment:
Do you have time to provide a screenshot of the render? We need to release this with "Cards" and "Buttons".

Reply:
I added some Shrine themed screenshots